AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -23,175 +25,223 @@
-		print(("\nError: Could not load file \"{0}\" in "
-			  "directory \"{1}\".\n".format('axonlist.npy', path_folder)))
-		raise
-	else:
-		return stats_array
-
-
-def display_axon_diameter(img,path_prediction,pred_axon,pred_myelin):
-	'''
+	"""
+    try:
+        stats_array = np.load(os.path.join(path_folder, "axonlist.npy"))
+    except IOError as e:
+        print(
+            (
+                '\nError: Could not load file "{0}" in '
+                'directory "{1}".\n'.format("axonlist.npy", path_folder)
+            )
+        )
+        raise
+    else:
+        return stats_array
+
+
+def display_axon_diameter(img, path_prediction, pred_axon, pred_myelin):
+    """
 	:param img: sample grayscale image (png)
 	:param path_prediction: full path to the segmented file (*_seg-axonmyelin.png) from axondeepseg segmentation output
 	:param pred_axon: axon mask from axondeepseg segmentation output
 	:param pred_myelin: myelin mask from axondeepseg segmentation output
 	:return: nothing
-	'''   
-	path_folder, file_name = os.path.split(path_prediction)
-	tmp_path = path_prediction.split('_seg-axonmyelin')
-
-	stats_array = get_axon_morphometrics(pred_axon,path_folder)
-	axon_diam_list = [d['axon_diam'] for d in stats_array]
-	axon_diam_array = np.asarray(axon_diam_list)
-	axon_iter = np.arange(np.size(axon_diam_array))
-
-	labels = measure.label(pred_axon)
-	axon_diam_display = a = np.zeros((np.shape(labels)[0], np.shape(labels)[1]))
-
-	for pix_x in np.arange(np.shape(labels)[0]):
-		for pix_y in np.arange(np.shape(labels)[1]):
-			if labels[pix_x,pix_y] != 0:
-				axon_diam_display[pix_x,pix_y] = axon_diam_array[labels[pix_x,pix_y]-1]
-	
-	# Axon overlay on original image + myelin display (same color for every myelin sheath)
-	plt.figure(figsize=(12,9))
-	plt.imshow(img, cmap='gray', alpha=0.8)
-	plt.imshow(pred_myelin,cmap='gray', alpha=0.3)
-	im = plt.imshow(axon_diam_display, cmap='hot', alpha=0.5)
-	plt.colorbar(im, fraction=0.03, pad=0.02)
-	plt.title('Axon overlay (colorcoded with axon diameter in um) and myelin display',fontsize=12)
-	plt.savefig(tmp_path[0] + '_map-axondiameter.png')
+	"""
+    path_folder, file_name = os.path.split(path_prediction)
+    tmp_path = path_prediction.split("_seg-axonmyelin")
+
+    stats_array = get_axon_morphometrics(pred_axon, path_folder)
+    axon_diam_list = [d["axon_diam"] for d in stats_array]
+    axon_diam_array = np.asarray(axon_diam_list)
+    axon_iter = np.arange(np.size(axon_diam_array))
+
+    labels = measure.label(pred_axon)
+    axon_diam_display = a = np.zeros((np.shape(labels)[0], np.shape(labels)[1]))
+
+    for pix_x in np.arange(np.shape(labels)[0]):
+        for pix_y in np.arange(np.shape(labels)[1]):
+            if labels[pix_x, pix_y] != 0:
+                axon_diam_display[pix_x, pix_y] = axon_diam_array[
+                    labels[pix_x, pix_y] - 1
+                ]
+
+                # Axon overlay on original image + myelin display (same color for every myelin sheath)
```

Comment:
I think the PIP8 conversion didn't work on some comments (at least this one), as this comment is not aligned with the code below (preceeded by too many tabs/spaces)

Reply:
Fixed!